<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="PC" failures="1" tests="1" name="批阅任务" time="37.871" errors="0" timestamp="26 12月 2018 11:25:38 GMT">
  <testcase classname="com.edu.teach_test.Piyue_Test" name="piyue" time="11.355">
    <failure type="java.lang.AssertionError" message="待验收错误 expected [0] but found [null]">
      <![CDATA[java.lang.AssertionError: 待验收错误 expected [0] but found [null]
at com.edu.teach_test.Piyue_Test.piyue(Piyue_Test.java:45)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- piyue -->
</testsuite> <!-- 批阅任务 -->
